modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/new-claim-button/index.html

```diff
@@ -0,0 +1,5 @@
+<div>
+	<button class="btn btn-primary newClaim">
```

Comment:
```suggestion
	<button class="btn btn-primary">
```

Se a classe nao existe no .css, nao precisar utilizar aqui

Reply:
a gente ta utilizando essa classe para o querrySelector do new-claim-button